x-pack/filebeat/module/misp/threat/test/misp-test.json.log-expected.json

```diff
@@ -0,0 +1,47 @@
+[
+    {
+        "@timestamp": "2017-03-30T12:54:26.000Z",
+        "event.category": "threat-intel",
+        "event.dataset": "misp.threat",
+        "event.module": "misp",
+        "event.type": "indicator",
+        "fileset.name": "threat",
+        "input.type": "log",
+        "log.offset": 0,
+        "service.type": "misp",
+        "threat.threat_indicator.attack_pattern": "[destination.ip = \"98.235.162.24\"]",
+        "threat.threat_indicator.description": "Tor exit nodes feed",
+        "threat.threat_indicator.feed": "misp",
+        "threat.threat_indicator.id": "58dcfe62-ed84-4e5e-b293-4991950d210f"
+    },
+    {
+        "@timestamp": "2017-03-30T12:54:26.000Z",
+        "event.category": "threat-intel",
+        "event.dataset": "misp.threat",
+        "event.module": "misp",
+        "event.type": "indicator",
+        "fileset.name": "threat",
+        "input.type": "log",
+        "log.offset": 460,
+        "service.type": "misp",
+        "threat.threat_indicator.attack_pattern": "[file.md5 = \"89357a1b2e32f2b9bddff94b8136810b\"]",
```

Comment:
Is this field value a MISP-specific DSL? Do we expect those to be used programmatically?

Do we need to translate to our field names (e.g. `file.hash.md5`)?

Reply:
No. This is STIX pattern expression, and it is meant to be used programmatically.